src/connectors/store.py

```
@@ -257,74 +234,52 @@ def flush(reason="batch"):
-    
-    return hybrid_retriever, all_docs
\ No newline at end of file
+
+
+def extract_topics(vectorstore: Qdrant):
+    manifest = VDBManifest(QDRANT_PATH)
+
+    # Add topics if needed
+    if not manifest.has_topics():
+        if manifest.num_chunks() > TOPIC_MIN_SIZE:
+            print(f"💾 Detectando temas...")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)